### PR TITLE
Update CHANGELOG.md for 2.29.0

### PR DESCRIPTION
**Pull request by shuoyang2016**

### CODE REVIEW

**Review by nareddyt — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Release 2.29.0 30-06-2021
+
+- Update API version to v10 (#547)
```
Comment (qiwzhang) [inline CHANGELOG.md:3]: remove line 3,  it is not useful for espv2 users.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- Disable preserve downstream scheme (#546)
+- Allow using another service account for GCS fetches. (#538)
```
Comment (qiwzhang) [inline CHANGELOG.md:5]: remove line 5 too

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# Release 2.29.0 30-06-2021
+
+- Update API version to v10 (#547)
+- Disable preserve downstream scheme (#546)
```
Comment (qiwzhang) [inline CHANGELOG.md:4]: line 4 change to:
restored the old way of setting scheme header according to upstream connection security level.

Comment (shuoyang2016) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (shuoyang2016) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (shuoyang2016) [reply CHANGELOG.md]: Done.

**Review by qiwzhang — APPROVED** [review]

**Review by TAOXUY — APPROVED** [review]

**Pull request merged**